### PR TITLE
initial version of new demo UI

### PR DESCRIPTION
**Pull request by UX3D-hohenester**

### CODE REVIEW

File: src/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-glslify": "^1.2.0",
     "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-scss": "^2.6.1",
```
Comment (UX3D-haertl) [inline src/package.json:36]: Do we need this plugin in the npm package?

File: ui.css [hunk 2]
```diff
@@ -0,0 +1,203 @@
+/* only contains styles for the ui */
```
Comment (UX3D-haertl) [inline ui.css:1]: Shouldn't this file be in the example folder?

File: src/package.json (hunk 1 above)
Comment (UX3D-becher) [reply src/package.json:36]: i don't think so, only in the example

Comment (UX3D-haertl) [reply src/package.json:36]: This seems to be in the wrong package.json. It is missing in the example

**Review by UX3D-becher — APPROVED** [review]

**Pull request merged**